src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -385,4 +363,56 @@ public boolean isAutoGrantPermissions() {
     public void setAutoGrantPermissions(boolean autoGrantPermissions) {
         this.autoGrantPermissions = autoGrantPermissions;
     }
+
+    public Platform getPlatformName() {
+        return platformName;
+    }
+
+    public String[] getCurrentSchemes() {
+        return currentSchemes;
+    }
+
+    public String getFullAppPath() {
+        if (appFile == null) {
+            return null;
+        }
+
+        Path path = Paths.get(appFile);
+        if (appFile.startsWith("sauce-storage:") || path.isAbsolute()) {
+            return appFile;
+        }
+        try {
+            URL url = new URL(appFile);
+            return appFile;
+        } catch(MalformedURLException e) {
+            // Ignore, this is expected to happen, parse as a regular path
+        }
+
+        return Paths.get(System.getProperty("user.dir"), path.normalize().toString()).normalize().toString();
+    }
+
+    public Map<String, Object> getCustomCapabilities() {
+        return customCapabilities;
+    }
+
+    public void setCustomCapabilities(Object keyCustomCapabilities) {
+        Map<String, Object> custom = new HashMap<>();
+        if (keyCustomCapabilities instanceof Map) {
```

Comment:
Because I want to support `String`, `Boolean` and Numbers, so a Map would be the only "not supported" type and it seemed easier to do the Map check and throw a custom exception than type check it and cast it multiple times.